packages/server/src/queries/data-services/transactions.ts

```diff
@@ -0,0 +1,25 @@
+import { apiClient } from "@osmosis-labs/utils";
+
+import { NUMIA_BASE_URL } from "../../env";
+import { Transaction } from "../complex/transactions/transaction-types";
```

Comment:
Ideally, this type should be defined with the query in this folder. IMO we should avoid circular dependencies with the complex folder. I'm considering moving it it the future (perhaps to a separate package). Keeping a clear separation with complex queries having a unidirectional dep => queries maintains this option.

Only the raw type would be here, the processed type for the frontend could remain in complex folder (that's a common pattern we have).